Utilities/macrecovery/macrecovery.py

```diff
@@ -457,7 +460,7 @@ def main():
                         help='Action to perform: "download" - performs recovery downloading,'
                         ' "selfcheck" checks whether MLB serial validation is possible, "verify" performs'
                         ' MLB serial verification, "guess" tries to find suitable mac model for MLB.')
-    parser.add_argument('-o', '--outdir', type=str, default=os.getcwd(),
+    parser.add_argument('-o', '--outdir', type=str, default='com.apple.recovery.boot',
                         help='customise output directory for downloading, defaults to current directory')
```

Comment:
It no longer defaults to current directory, thus this should be renamed.

Reply:
Yeah sorry, commit dreamwhite/OpenCorePkg@24d83a3 has the fixed help message